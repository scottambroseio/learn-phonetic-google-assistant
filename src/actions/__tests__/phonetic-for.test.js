// @flow

import phoneticFor from '../phonetic-for';

describe('phonetic-for', () => {
  it('should generate a correct response', () => {
    const appMock = { tell: jest.fn(), getArgument: () => 'b' };
    const appMock2 = { tell: jest.fn(), getArgument: () => 'C' };

    phoneticFor(appMock);
    phoneticFor(appMock2);

    expect(appMock.tell.mock.calls[0][0]).toMatchSnapshot();
    expect(appMock2.tell.mock.calls[0][0]).toMatchSnapshot();
  });
});
