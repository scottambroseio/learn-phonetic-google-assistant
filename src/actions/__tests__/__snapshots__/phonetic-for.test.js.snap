// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`phonetic-for should generate a correct response 1`] = `"The phonetic for the letter b is bravo"`;

exports[`phonetic-for should generate a correct response 2`] = `"The phonetic for the letter c is charlie"`;
